src/app/globals.css

```diff
@@ -25,3 +25,28 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+h1 {
+  font-size: 2em;
+  margin-bottom: 1rem;
+}
+
+h2 {
+  font-size: 1.5em;
+  margin-bottom: 0.5rem;
+}
+
+p {
+  margin-bottom: 0.5rem;
+}
+
+.centerText {
+  text-align: center;
+}
+
+.page-landing-background {
+  background-image: url("/ssd-background.jpg");
```

Comment:
[Please Change]: There's two other helpful properties to have with background images like this:

```css
background-position: center
background-size: cover
```

Using `center` keeps the image centered (as opposed to being top-left), while `cover` allows auto-scaling the image to fit the whole image on the screen.

Reply:
Done.